presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -230,6 +230,8 @@ public PlanOptimizers(
         PlanOptimizer predicatePushDown = new StatsRecordingPlanOptimizer(optimizerStats, new PredicatePushDown(metadata, sqlParser));
 
         builder.add(
+                // Clean up all other symbols or expressions which are not a part of the given query
+                new PruneUnreferencedOutputs(),
```

Comment:
can you use `columnPruningRules`

Reply:
Yes. I will replace it with `IterativeOptimizers` but some tests were failed when we rearrange the optimizers. So will identify that and update things.